lib/containers/container_images.pm

```diff
@@ -201,11 +201,12 @@ sub test_zypper_on_container {
     die 'Argument $image not provided!' unless $image;
     die 'Argument $runtime not provided!' unless $runtime;
 
-    $runtime->run_container($image, cmd => "zypper lr -s", keep_container => 1, timeout => 120);
-    $runtime->run_container($image, name => 'refreshed', cmd => "zypper -nv ref", keep_container => 1, timeout => 300);
+    record_soft_failure("bsc#1192941 zypper-docker entrypoint confuses program arguments") if (script_run("docker run --rm -ti $image zypper ls | grep 'Usage:'") == 0);
```

Comment:
```suggestion
    record_soft_failure("bsc#1192941 zypper-docker entrypoint confuses program arguments") if (script_run("$runtime run --rm -ti $image zypper ls | grep 'Usage:'") == 0);
```

Reply:
Good that you point this out. The error is a different one: This should on run for `docker` .....